doc/changelog/04-tactics/18395-master+use-intro-name-in-evar-context-when-goal-is-evar.rst

```diff
@@ -0,0 +1,10 @@
+- **Changed:**
+  :tac:`intro z` on an existential variable goal forces the resolution
```

Comment:
`:tac:` doesn't seem to be a proper tag. Since it's not a tactic reference either I think you can just write ``intro z``.

Reply:
OK, I made another try.